### ISSUE TITLE
Adjust language button to clarify region

### ISSUE THREAD

**Issue by amycole501** [GSA/usagov-benefits-eligibility#718]
In JAWS the language button says it's both a banner region and a navigation region. I'm not sure which it should be but I think it's banner. May warrant a minor fix. 

The header does now read correctly as "header primary navigation region".

_Originally posted by @amycole501 in https://github.com/GSA/usagov-benefits-eligibility/issues/581#issuecomment-1498177369_

**Comment by amycole501** [comment]
https://github.com/GSA/usagov-benefits-eligibility/assets/117778073/6fe1b298-679f-4dfa-84d6-e21d047f431e

I recorded tabbing to the language button on the live BEARS and also the live/current version of the USA site to compare the two. On the BEARS site you can hear "banner region; navigation region" and no indication that there's a list with an item. (personally, I think it needs to be a button declaring which state the page is in but that'd be a different discussion for later).

**Comment by EnriqueOrt** [comment]
@amycole501 - Could you provide some more clarification on this issue and give it a rating to know whether this is something that absolutely needs to be adjusted or if we can push it to a later sprint planning?

**Comment by amycole501** [comment]
It can be pushed; it's confusing but not a show-stopper.
